geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionStatsDistributedTest.java

```diff
@@ -220,45 +252,48 @@ private void putDataInRegion(final String regionName) {
   }
 
   private void createPartitionedRegionWithRedundantCopies(final String regionName) {
-    PartitionAttributesFactory paf = new PartitionAttributesFactory();
+    PartitionAttributesFactory<?, ?> paf = new PartitionAttributesFactory<>();
```

Comment:
Rather than `<?, ?>`, could this test parameterize things in a concrete way? Looking at the tests, it seems like the Region always uses `<Integer, Integer>`.

Reply:
Made concrete